### DISCUSSION TITLE
Right-to-left language support

### DISCUSSION THREAD

**Discussion by MusiCode1** [Budibase/budibase#6145] [Help]
Peace.
The platform looks great.
But as a right-to-left language speaker, I have not been able to adapt the interface to my lessons.
Is there support for right-to-left alignment?

[Translated via Google Translate, sorry]

**Comment by melohagan** [comment]
Hi @MusiCode1 

I'm not quite sure exactly what you mean, but you can get the characters to read right-to-left with a little <a href="https://docs.budibase.com/docs/custom-css">Custom CSS</a>

```
direction: rtl;
unicode-bidi: bidi-override;
```

![Screenshot 2022-06-01 at 08 14 35](https://user-images.githubusercontent.com/101575380/171348607-d2512905-f585-4fad-8638-7a64c58dc0b1.png)

**Reply by dodyg** [reply]
They are talking about proper RTL support when the whole layout flows properly. It is critical for languages such as Arabic, Persian or Hebrew. Without RTL support, budibase cannot be used for those language speakers. 

People can contribute to translations easily but RTL support will have to come from expert builders in this system.
